### PR TITLE
Update es6-destructuring.md

### PR DESCRIPTION
**Pull request by RajaBellebon**
@olajohn-ajiboye this blog is 🏆, I just fixed couple of sentences and typos!

### CODE REVIEW

File: content/post/es6-destructuring.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ We'll look at this in two parts, **array** and **object** destructuring in progr
 
 * **Array destructuring :boom:**
 
-> Let's say you have an array of of my profile.
```
Comment (RajaBellebon) [inline content/post/es6-destructuring.md]: Maybe we can remove the `my` too?

**Comment by RajaBellebon** [comment]
Not sure why the deployment failed maybe because I am on a fork :).

**Comment by olajohn-ajiboye** [comment]
> Not sure why the deployment failed maybe because I am on a fork :).

It's all fine. It is no longer hosted on Netifly. I now have it hosted with Firebase. So, the deployment fail isn't relevant.

File: content/post/es6-destructuring.md (hunk 1 above)
Comment (olajohn-ajiboye) [reply content/post/es6-destructuring.md]: Yea, I think it makes sense to remove it. Since this isn't even my real profile.

**Pull request merged**